src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx

```diff
@@ -170,10 +199,38 @@ export class LdSidenavNavitem implements InnerFocusable {
               {this.abbreviation}
             </span>
           )}
+
+          <ld-tooltip
+            show-delay="250"
+            tag="span"
+            ref={(el) => (this.tooltipRef = el)}
+            class="ld-sidenav-navitem__tooltip"
+            disabled={!this.sidenavCollapsed}
+            arrow
+            position={
+              this.sidenavAlignement === 'left' ? 'right middle' : 'left middle'
+            }
+          >
+            <div
+              class="ld-sidenav-navitem__tooltip-trigger"
+              slot="trigger"
+              onClick={this.onClick}
+            />
```

Comment:
Why don't you just use the whole component (i.e. the `<Tag>` element) as trigger?

Reply:
I tried that. It destroyed the whole layout. Many side effects. Using an independent element was way less destructive.